develop.html.mako

```diff
@@ -27,22 +27,40 @@ on getting started.
 <a name="developers"></a>
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where do you find the developers ?  Four places:</p>
 
 <ul>
-    <li>The <a href="/support.html#mailinglist">main
-    mailing list</a> is the most common - core devs assist users with issues of all kinds.</li>
-    <li>Developer discussion takes place
-    on the <a href="http://groups.google.com/group/sqlalchemy-devel">development mailing list</a>,
-    also hosted on Google groups.  Anyone can join this
-    list, though we ask that discussion be limited to architectural and developmental issues only.
-    </li>
-    <li>Some development discussion occurs on the
-    <code>#sqlalchemy-devel</code> IRC channel on Freenode.
-    This channel is <strong>only</strong> for users that are interested in contributing code,
-    tests, documentation, or other developmental resources, and no support issues will be
-    addressed here.   IRC-based support is available on the
-    <a href="/support.html#irc">main IRC channel</a>.</li>
+    <li>The <a href="/support.html#mailinglist">main mailing list</a> is the most 
+        common - core devs assist users with issues of all kinds.
+        </li>
+    <li>Developer discussion takes place on the 
+        <a href="http://groups.google.com/group/sqlalchemy-devel">development
+        mailing list</a>, also hosted on Google groups.  Anyone can join this list, 
+        though we ask that discussion be limited to architectural and developmental 
+        issues only.
+        </li>
+    <li>Some development discussion occurs on the <code>#sqlalchemy-devel</code>
+        IRC channel on Freenode. This channel is <strong>only</strong> for users 
+        that are interested in contributing code, tests, documentation, or other 
+        developmental resources.
+        <strong>No support issues will be addressed here.</strong>
+        IRC-based support is available on the <a href="/support.html#irc">main 
+        IRC channel</a>.
+        </li>
+    <li>A real-time virtual meeting of core developers and contributers occurs in the
+        <a href="https://gitter.im/sqlalchemy/devel">sqlalchemy/devel</a> gitter room
+        on a regular basis. This meeting is intended to be held weekly at a regular time,
+        but this may vary based on the availability of required members.  In advance of 
+        every meeting, the room will contain a link to a document containing the Date, 
+        Time and Agenda for the next scheduled meeting; currently this appears in the 
+        "Subject Line" or "Title"  of the room. These meetings and their records are 
+        open to the public; users who are interested in contributing code, tests, 
+        documentation, or other developmental resources are encouraged to attend.
+        <strong>No support issues will be addressed here.</strong>
```

Comment:
let's be optimistic to start (ha) and not include any of this second part of the paragraph for now (everything from the `<strong>` tag on forward).